DiscTests/Networking/Authorization/CreateLoginUrlSpec.swift

```diff
@@ -67,7 +61,7 @@ class CreateLoginUrlSpec: QuickSpec {
                 it("should result in login URL") {
                     let scopes: [Scope] = [ .ContentManagement, .UpdateProfile ]
                     let provider: Provider = .GitHub
-                    let url = oauth.createLoginUrl(scopes: scopes, provider: provider)
+                    let url = APIClient.createLoginUrl(clientId: clientId, redirectUri: redirectUri, scopes: scopes, provider: provider)
                     
                     let expectedUrlString = "https://passport.thegrid.io/login/authorize/\(provider.rawValue)"
```

Comment:
Is this correct? 
`return createRequest(.POST, "/api/auth/login", body: body)` is the correct URL